src/java/com/google/cloud/bigquery/dwhassessment/hooks/testing/BUILD

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+load("@rules_java//java:defs.bzl", "java_library")
+
+package(default_visibility = ["//src:internal"])
+
+java_library(
+    name = "testing",
+    srcs = glob(["*.java"]),
+    testonly = 1,
+    deps = [
+        "//src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger",
+        "@maven//:org_apache_avro_avro",
```

Comment:
I see that some imports still use `@maven`. I think it works only because other test files also have dependencies to the same packages, but defined in `@maven_tests` – packages in `@maven` are not available in runtime.